swarm/tools/validation/validators/prompts.py

```
@@ -0,0 +1,127 @@
+        if filename_key not in registry:
+            continue
+
+        agent_meta = registry[filename_key]
+        if agent_meta.get("source") != "project/user":
+            continue
+
+        try:
+            content = path.read_text(encoding="utf-8")
+        except Exception:
+            # Skip files that can't be read (already reported elsewhere)
+            continue
```

Ruff (0.14.13)

`try`-`except`-`continue` detected, consider logging the exception

(S112)